src/test/java/com/cloudant/tests/SslAuthenticationTest.java

```diff
@@ -162,9 +164,7 @@ public static void tearDownClass() {
 
     @After
     public void tearDown() {
-        if (dbClient != null) {
-            dbClient.shutdown();
-        }
+        dbClient.shutdown();
```

Comment:
Strangely this test does not configure the `dbClient` in a `@Before` method, but it does appear to new up `dbClient` in each test. I do think that there is the possibility for a test to fail without setting `dbClient` though so I think the null check is worth having at least until we refactor this test with the JUnit `Rule`.